src/Components/NavBar/Menus/DropDownSection.tsx

```diff
@@ -0,0 +1,88 @@
+import { Box, color, Flex, MenuItem, Sans } from "@artsy/palette"
+import React from "react"
+import { MenuLinkData, SimpleLinkData } from "../menuData"
+
+interface DropDownSectionProps {
+  section: MenuLinkData
+}
+
+export const DropDownSection: React.FC<DropDownSectionProps> = ({
+  section,
+}) => {
+  if (!section) return null
+
+  return (
+    <>
+      {section.text === "Artist Nationality & Region" ? (
```

Comment:
This bit of logic here is to achieve the layout outlined in the spec.

<img width="465" alt="Screen Shot 2020-04-14 at 12 39 04 PM" src="https://user-images.githubusercontent.com/5201004/79250698-12ba4400-7e4d-11ea-983b-8195c6c11cc7.png">